.github/workflows/wait_for_required_workflows.yml

```diff
@@ -65,12 +65,14 @@ jobs:
                   actions = actions.filter(action => action !== item.name)
                 }
               })
-              await new Promise(r => setTimeout(r, 1000));
+              if (actions.length === 0) {
+                console.log("No actions to wait for")
+                return
+              }
+              
+              await new Promise(r => setTimeout(r, 5000));
```

Comment:
I think 1 second polling is a bit too much, so changed it to 5 seconds to also avoid hitting rate limits